examples/hub_spoke_common/example.tfvars

```diff
@@ -2,7 +2,7 @@ project_id      = null
 public_key_path = "~/.ssh/gcp-demo.pub"
 region          = "us-east1"
 fw_image_name   = "vmseries-flex-bundle2-1010"
-fw_machine_type = "n1-standard-4"
+fw_machine_type = "n2-standard-4"
```

Comment:
We need clarify it - from what version of PanOS we can use n2 - vmseries 10.1 n1 is recommended <=Skylake platform.

https://docs.paloaltonetworks.com/vm-series/10-1/vm-series-performance-capacity/vm-series-performance-capacity/vm-series-on-google-performance-and-capacity

Reply:
n2 is fully supported on 10.1 too